Class Presentations/reveal-presentations/SelfConf2016.html

```diff
@@ -0,0 +1,582 @@
+				<img src="../presentation-images/selfconf2016/reveal_js.jpg" width="320px" height="156px">
+				<aside class="notes">Reveal's added benefit: practice using JS</aside>
+				</section>
+
+				<section>
+				<h3>For teaching the course</h3>
+				<ul>
+					<li>Mr. Lincoln, engineering teacher extraordinaire</li>
+				</ul>
+				<aside class="notes">Really was helpful to have him with the students</aside>
+				</section>
+
+				<section>
+					<h2>Goals</h2>
+					<ul class="checklist">
+						<li>
+							<input type="checkbox" id="goal0" checked />
+							<label for="goal0">Who Are We, and What Is This Project?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal1" checked />
+							<label for="goal1">Why Did We Decide to Do This?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal2" checked />
+							<label for="goal2">What Resources Did We Use?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal3" />
+							<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal4" />
+							<label for="goal4">What Are Some Things We Didn't Think About?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal5" />
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+
+
+
+
+
+
+
+
+				<section>
+					<h2>What Has Been Easy? Difficult? Fun?</h2>
+				</section>
+
+				<section>
+				<h3>Remembering the resource gap and checking our privilege--difficult.</h3>
+				<aside class="notes"> not everyone has a computer they can tinker on at
+					 home, so you need paper-friendly resources as well.
+				</aside>
+				</section>
+
+				<section>
+				<h3>Forming relationships with the students--easy...</h3>
+				<aside class="notes"> Easy because we laid a good foundation by
+					 dedicating the first two sessions to getting to know each other and
+					 setting clear expectations.
+				</aside>
+				</section>
+
+				<section>
+				<h3>Keeping the lessons engaging, and trying to teach to multiple
+					learning styles--difficult.</h3>
+				<aside class="notes"> The students gave us new ideas throughout the class,
+					though, as well as through a survey we sent out at the halfway point of
+					the program.
+				</aside>
+				</section>
+
+				<section>
+				<h3>The challenge of keeping up with the students growth and development
+					--fun.</h3>
+				<aside class="notes"> It made us better teachers, as well.
+				</aside>
+				</section>
+
+				<section>
+				<h3>Scheduling--difficult.</h3>
+				<aside class="notes"> Things always change, and people are busy:
+					after-school activities, state testing that ends up needing to
+					 be redone...
+				</aside>
+				</section>
+
+				<section>
+				<h3>Watching the students light up when their code works--fun.</h3>
+				<aside class="notes">
```

Comment:
Add: This is a personal high-light of mine. Helping guide someone to "getting it" has been very fulfilling.